tasks/filter-team-data/test/test_team_data_filterer.rb

```diff
@@ -13,7 +13,7 @@
       ]
 
       results = TeamDataFilterer.transform_links(links)
-      expect(results).must_equal([
+      results.must_equal([
```

Comment:
Hmm, was working for me: http://docs.seattlerb.org/minitest/Minitest/Spec/DSL/InstanceMethods.html#method-i-expect